packages/react-native/Libraries/Text/TextInput/RCTBaseTextInputShadowView.m

```diff
@@ -218,7 +218,22 @@ - (NSAttributedString *)measurableAttributedText
 
 - (CGSize)sizeThatFitsMinimumSize:(CGSize)minimumSize maximumSize:(CGSize)maximumSize
 {
-  NSAttributedString *attributedText = [self measurableAttributedText];
+  NSMutableAttributedString *attributedText = [[self measurableAttributedText] mutableCopy];
```

Comment:
As far as I remember we need to sometimes add characters otherwise the size is not computed properly. 
For instance when we have multiple empty lines. We do it only for layout part and it's not visible on the screen.

Reply:
Ah I think on iOS we only add newlines but still we need to modify the object.